include/wx/setup_inc.h

```diff
@@ -757,7 +757,7 @@
 
 // notice that we can't use wxCHECK_VISUALC_VERSION() here as this file is
 // included from wx/platform.h before wxCHECK_VISUALC_VERSION() is defined
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
Oops, we indeed have a problem here. We should normally include `wx/msw/gccpriv.h` right after `wx/compiler.h` (because it's also related to compiler-specific checks/stuff), but we can't include it before deciding on the value of `wxUSE_UNICODE`, so we can't include it before `wx/setup.h`...

I guess we could try doing this anyhow and also move the "adjust Unicode setting" block of code further up and rely on `UNICODE` or `_UNICODE` being defined for Unicode mode builds. But while this probably should work in practice, it seems wrong to require something special to be defined when Unicode is the default.

The simplest solution I see is probably to forcibly set this option (and also `wxUSE_DBGHELP`, which suffers from the same problem) to 0 in `wx/msw/chkconf.h` if `__MINGW32_TOOLCHAIN__` is defined, as we already do for `wxUSE_SECRETSTORE`, for example. Let's do this for now.